.github/workflows/copy-workflow.yml

```diff
@@ -23,14 +23,33 @@ jobs:
         repository: ${{ matrix.cfg.target }}
         token: ${{ secrets.WEB3BOT_GITHUB_TOKEN }}
         persist-credentials: true
+    - name: git config
+      run: |
+        git config user.name web3-bot
+        git config user.email "web3-bot@users.noreply.github.com"
+    - name: run go mod tidy
+      run: |
+        go mod edit -go 1.15
+        go mod tidy
+        if ! git diff --quiet; then
+          git add .
+          git commit -m "run go mod tidy"
+          echo "NEEDS_UPDATE=1" >> $GITHUB_ENV
+        fi
+    - name: gofmt
+      run: |
+        gofmt -s -w .
+        if ! git diff --quiet; then
+          git add .
+          git commit -m "run gofmt -s"
+          echo "NEEDS_UPDATE=1" >> $GITHUB_ENV
```

Comment:
I kinda wonder if you could later just ask git "are there any commits ahead of the upstream branch?" instead of using NEEDS_UPDATE. Unrelated to this change though.

Reply:
As we're committing to master here, we could just check if HEAD != origin/master. Just need to figure out an elegant way to pull that out of git.

Reply:
Will do that in a separate PR.

Reply:
Sounds good :) all LGTM